packages/expo-image/android/build.gradle

```diff
@@ -73,15 +73,18 @@ repositories {
     url "$rootDir/../node_modules/jsc-android/dist"
   }
   google()
-  jcenter()
+  mavenCentral()
 }
 
 dependencies {
+  def GLIDE_VERSION = "4.12.0"
```

Comment:
I'm not sure I'm a fan of this approach 🤔 
Give me some arguments for extracting the library version to a variable 🤔 
My argument that counters this approach is: `in package.json you write every library version next to the library name` (yeah, I know json does not allow variables but still...) 🤔

Reply:
I'm a fan of parametrizing things that should be together - here we have a base glide library in some version that has to be in sync with annotation processor and plugin version. That's exactly the use-case for variables, allowing changing contents everywhere by doing it in one place. It's local as well so I don't see any problem here.